tests/lsdb/catalog/test_polygon_search.py

```diff
@@ -56,23 +56,29 @@ def test_polygon_search_invalid_shape(small_sky_order1_catalog):
 def test_polygon_search_wrapped_right_ascension():
     """Tests the scenario where the polygon edges intersect the
     discontinuity of the RA [0,360] degrees range. For the same
-    polygon we can have four possible combination of coordinates
-    (with some float-fudging):
-        - [(-20, 1), (-20, -1), (20, -1), (20, 1)]
-        - [(-20, 1), (-20, -1), (380, -1), (380, 1)]
-        - [(340, 1), (340, -1), (20, -1), (20, 1)]
-        - [(340, 1), (340, -1), (-340, -1), (-340, 1)]
-    """
+    polygon we have several possible combination of coordinates
+    (here with some float-fudging)."""
+    def get_vertex_combinations(vertex):
+        ra_combinations = (vertex[0], vertex[0] + 360, vertex[0] - 360)
+        return [(ra, vertex[1]) for ra in ra_combinations]
+
+    def calculate_polygon_combinations(vertices):
```

Comment:
This is more thorough, but I think the explicit list makes it easier to understand what's being tested (and I can kind of reason about the math when I see some real numbers).

Reply:
Thanks, makes sense to me - just added the explicit values but reduced the polygon to a triangle so that we have fewer combinations to copy-paste!